src/mds/MDLog.h

```diff
@@ -90,6 +90,7 @@ class MDLog {
     ReplayThread(MDLog *l) : log(l) {}
     void* entry() {
       log->_replay_thread();
+      detach();
```

Comment:
If I recall correctly, calling detch() on a thread makes it unsafe to later call join() on it.  We call join on replay_thread in MDLog::shutdown.

Maybe instead of detaching, we should be joining the thread when MDSRank does a standby_replay_restart?